api/database/sampleApp.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
+from mongoengine import *
```

Comment:
I don't think importing everything is a good idea, I think we should either import specific functions or just import it and use the mongoengine.xxx syntax